lib/input/check_headers.py

```diff
@@ -7,278 +7,277 @@
 from lib.input.imap.header import get_mail_header
 from lib.util.read import lines_per_n
 
+class CheckHeaders(object):
 
-def get_unavailable_uid():
-	"""
+	def get_unavailable_uid(self):
+		"""
 
-	This function returns a list of UIDs that are not available in the IMAP server.
+		This function returns a list of UIDs that are not available in the IMAP server.
 
-	:return: List containing the UIDs not available in the IMAP server
-	"""
-	imaplib._MAXLINE = 800000
-	conn = open_connection()
-	conn.select('INBOX')
-	search_str = 'UID ' + '1:*'
-	retcode, uids = conn.uid('SEARCH', None, search_str)
-
-	available_uid = []
-	for uid in uids[0].split():
-		available_uid.append(int(uid))
-
-	try:
-		conn.close()
-	except:
-		pass
-	conn.logout()
+		:return: List containing the UIDs not available in the IMAP server
+		"""
+		imaplib._MAXLINE = 800000
+		conn = open_connection()
+		conn.select('INBOX')
+		search_str = 'UID ' + '1:*'
+		retcode, uids = conn.uid('SEARCH', None, search_str)
 
-	return set(range(min(available_uid), max(available_uid)+1)) - set(available_uid)
+		available_uid = []
+		for uid in uids[0].split():
+			available_uid.append(int(uid))
 
-# This list stores the UIDs of mails that have duplicate entries in the JSON file.
-duplicate_uid = set()
+		try:
+			conn.close()
+		except:
+			pass
+		conn.logout()
 
-# This set stores the UIDs of mails that don't have an entry in the JSON file - UIDs are consecutive numbers.
-missing_uid = set()
+		return set(range(min(available_uid), max(available_uid)+1)) - set(available_uid)
 
-# This list stores the UIDs of mails that have entries with insufficient entries in the JSON file.
-invalid_uid = set()
+	# This list stores the UIDs of mails that have duplicate entries in the JSON file.

```

Comment:
These variable declarations can be moved to top and sphinx documentation created for the class.